en/docs/learn/tutorials/edit-an-api-by-modifyng-the-api-definition.md

```diff
@@ -0,0 +1,213 @@
+
+In this tutorial, let's see how you can add interactive documentation to an API by directly editing the Swagger code via the API Publisher UI.
+
+!!! note
+    This tutorial uses the `PizzaShack` API created in [Create a REST API]({{base_path}}/learn/design-api/create-api/create-a-rest-api/) section and Published in the [Publish an API]({{base_path}}/learn/design-api/publish-api/publish-an-api/) section.
+ .
+
+
+1.  Sign in to the API Publisher and choose to design a new REST API. `https://<hostname>:9443/publisher`
+
+    ![create_a_rest_api]({{base_path}}/assets/img/learn/create-a-rest-api.jpg)
+
+
+2.  Click **Design a New REST API** .
+
+3.  In the **Create an API** page, give an API name, a context a version and the endpoint as follows and click **Create**. 
+
+    <table><colgroup> <col/> <col/> <col/> </colgroup><tbody><tr><th colspan="2" >Field</th><th >Sample value</th></tr><tr><td colspan="2" class="confluenceTd">Name</td><td class="confluenceTd">PizzaShack</td></tr><tr><td colspan="2" class="confluenceTd">Version</td><td colspan="1" class="confluenceTd">1.0.0</td></tr><tr><td colspan="2" class="confluenceTd">Context</td><td class="confluenceTd"><div class="content-wrapper"><p><code>/pizzashack</code></p><div><div class="confluence-information-macro-body"></div><div class="confluence-information-macro confluence-information-macro-tip"><span class="aui-icon aui-icon-small aui-iconfont-approve confluence-information-macro-icon"></span><div class="confluence-information-macro-body"></div></div></div></div></td></tr><tr><td colspan="2" class="confluenceTd">Endpoint</td><td colspan="1" class="confluenceTd"><p><a class="external-link" href="https://localhost:9443/am/sample/pizzashack/v1/api" rel="nofollow">https://localhost:9443/am/sample/pizzashack/v1/api/</a></p><p>The endpoint that you add is automatically added as the production and sandbox endpoints.</p></td></tr></tbody></table>
+         
+    ![create_an_API]({{base_path}}/assets/img/learn/tutorials/create-a-rest-api-pizzashack.png)
+
+    The **Overview** of the created API will be displayed. 
+
+    ![create_a_rest_api]({{base_path}}/assets/img/learn/overviewpage-rest-api.jpg)
+
+
+4. Click on **API definition** to view the API Definition in the swagger UI.
+    The Swagger UI opens.
+    ![]({{base_path}}/assets/img/learn/tutorials/rest-api-definition-pizzashack.png)
+  
+5. Add the following GET and PUT method for the API.
+       1. Under the `paths` object, remove the `{}` and add the following code, as shown in the screenshot below.
+
+        !!! tip
+            In the code below, note that you have a resource defined with the URL pattern `/order/{orderId}` under the `paths` object. This is followed by the HTTP methods GET and PUT. For each HTTP method, the following parameters are defined.
+            
+            -   **responses:** An object to hold responses that can be used across operations. See the Swagger specification for details. **
+            **
```

Comment:
The ** ** at the end of the sentence on lines 46 and 47 are a mistake as it is not bolding anything.